cou/utils/openstack.py

```diff
@@ -83,25 +90,39 @@ def _generate_lookup(cls, resource: str) -> OrderedDict:
         :return: Ordered dictionary containing the version and the compatible OpenStack release.
         :rtype: OrderedDict
         """
-        with open(resource) as csv_file:
+        with open(resource, encoding=encodings.utf_8.getregentry().name) as csv_file:
```

Comment:
Why we are not using simply "UTF-8" here? What are the benefits?

Reply:
If you open without encoding pylint warns. Thus I checked the file it was utf-8 so I used utf-8

Reply:
Sorry I misunderstood your question: If there is a typed alternative or constant I prefer to use it.

Reply:
What I meant is to use `with open(resource, encoding="utf-8") as csv_file:` since this is well know name we do not need to use such long way to get it.
Now I realized that you are missing `mode="r"`.

Reply:
to me it's same as if you use `"rb"` (mode) as constant, but it's up to you.